[Project] Establish a Lyft support rotation (#2051)
envoyproxy/envoy-mobile: docs/Support.md

# Support Rotation

Envoy Mobile has a support rotation system to ensure that discussions,

> issues and pull requests are handled in a timely manner from someone who
> works on Envoy Mobile at Lyft.

Reviewer: ```suggestion
issues and pull requests are handled in a timely manner from an Envoy Mobile maintainer.
```

Just because I'm an optimist: I think we can avoid using Lyft grouping so when our current friends are well onboarded and can help participate without being part of Lyft

Author: Some of the friends of Envoy Mobile have explicitly requested a rotation like this so they can reach someone within Lyft.

I'd love for there to be a more general Envoy Mobile rotation, and remove the need in having feedback from a specific company, at some point in the near future, at which point we can and should generalize the wording here.

## Who's this week's current support maintainer?

The current support maintainer's GitHub username is defined
[here](https://github.com/envoyproxy/envoy-mobile/blob/main/.github/lyft_maintainers.yml#L1).

## Responsibilities

* **Monitor GitHub Issues & Pull Requests:** Ensure that discussions on
  GitHub are engaged with in a prompt manner.
* **Monitor Slack:** Engage with Slack discussions in the Envoy slack
  org ([#envoy-mobile](https://envoyproxy.slack.com/archives/CKQ2LK23G),
  [#envoy-mobile-engflow-ci](https://envoyproxy.slack.com/archives/C02QMNG92A3)).

Even though there is no concrete SLA, the support maintainer will make a
reasonable effort to engage with discussions within 1 business day.

## Why is the rotation limited to Lyft employees?

Envoy Mobile is a project maintained by contributors from multiple
companies, as highlighted in [OWNERS.md](../OWNERS.md), but as the
project grows we believe it will be helpful to have a point of contact

> within a company that has bee using Envoy Mobile in production.

Reviewer 2: nit bee -> been
